packages/grz-common/src/grz_common/constants.py

```diff
@@ -1,40 +1,7 @@
 """Constants for logging configuration, JSON schema validation, and other settings."""
 
-# This will be overridden by the package that imports it
-PACKAGE_ROOT = "grz_common"
-
 LOGGING_FORMAT = "%(asctime)s [%(levelname)s] %(name)s: %(message)s"
 LOGGING_DATEFMT = "%Y-%m-%d %I:%M %p"
```

Comment:
These don't really need to be in `constants`. Can be in `logging_setup.py`, side node: rename to `logging.py`?